xcp/logger.py

```diff
@@ -71,11 +73,13 @@ def openLog(lfile, level=logging.INFO):
     handler.setFormatter(FORMAT)
     handler.setLevel(level)
     LOG.addHandler(handler)
+    our_handlers.append(handler)
     return True
 
 def closeLogs():
     """Close all logs"""
-    for h in LOG.handlers:
+    for h in our_handlers:
```

Comment:
Modifying the list during iteration doesn't work as expected. It skips elements.

You can do something like this instead:

```
while our_handlers:
    h = our_handlers[0]
    ...
```

Reply:
So it looks like this may have been buggy before as well since `LOG.handlers` is just a list so it has the same skipping behaviour for its iterator. Oh well, now is a good time to fix it...

Reply:
Openned #10 for this one